package.json

```diff
@@ -14,6 +14,15 @@
   "license": "ISC",
   "dependencies": {
     "@json-schema-tools/meta-schema": "^1.7.0",
+    "@opentelemetry/api": "^1.7.0",
+    "@opentelemetry/exporter-metrics-otlp-proto": "^0.48.0",
+    "@opentelemetry/instrumentation-fastify": "^0.33.0",
+    "@opentelemetry/instrumentation-http": "^0.48.0",
+    "@opentelemetry/instrumentation-pino": "^0.35.0",
+    "@opentelemetry/resources": "^1.21.0",
+    "@opentelemetry/sdk-metrics": "^1.21.0",
+    "@opentelemetry/sdk-node": "^0.48.0",
+    "@opentelemetry/semantic-conventions": "^1.21.0",
```

Comment:
Quite a few deps. Do we reference these all directly?

Reply:
I think the fastify one I ended up not being happy with